2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'meme_experiments', 'entity': 'vietnguyen', 'root_dir': '/content/SSLMemes', 'sweep_id': '72jewzua', 'run_id': 'gtgb5bqw', 'sweep_param_path': '/content/SSLMemes/wandb/sweep-72jewzua/config-gtgb5bqw.yaml'}
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'src/train.py', 'program': '/content/SSLMemes/src/train.py'}
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_init.py:_log_setup():405] Logging user logs to /content/SSLMemes/wandb/run-20221011_043400-gtgb5bqw/logs/debug.log
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_init.py:_log_setup():406] Logging internal logs to /content/SSLMemes/wandb/run-20221011_043400-gtgb5bqw/logs/debug-internal.log
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_init.py:init():439] calling init triggers
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_init.py:init():443] wandb.init called with sweep_config: {'batch_size': 64, 'dropout': 0.4, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.07480114716579632, 'optimizer': 'adam'}
config: {}
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [wandb_init.py:init():492] starting backend
2022-10-11 04:34:00,855 INFO    Thread-2  :30414 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-11 04:34:00,872 INFO    Thread-2  :30414 [backend.py:ensure_launched():219] starting backend process...
2022-10-11 04:34:00,882 INFO    Thread-2  :30414 [backend.py:ensure_launched():225] started backend process with pid: 30447
2022-10-11 04:34:00,884 INFO    Thread-2  :30414 [wandb_init.py:init():501] backend started and connected
2022-10-11 04:34:00,893 INFO    Thread-2  :30414 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 64, 'dropout': 0.4, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.07480114716579632, 'optimizer': 'adam'}
2022-10-11 04:34:00,895 INFO    Thread-2  :30414 [wandb_init.py:init():565] updated telemetry
2022-10-11 04:34:00,923 INFO    Thread-2  :30414 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-10-11 04:34:01,931 INFO    Thread-2  :30414 [wandb_run.py:_on_init():1759] communicating current version
2022-10-11 04:34:01,974 INFO    Thread-2  :30414 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-11 04:34:01,974 INFO    Thread-2  :30414 [wandb_init.py:init():625] starting run threads in backend
2022-10-11 04:34:04,827 INFO    Thread-2  :30414 [wandb_run.py:_console_start():1733] atexit reg
2022-10-11 04:34:04,828 INFO    Thread-2  :30414 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-10-11 04:34:04,828 INFO    Thread-2  :30414 [wandb_run.py:_redirect():1611] Redirecting console.
2022-10-11 04:34:04,829 ERROR   Thread-2  :30414 [wandb_run.py:_redirect():1670] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_run.py", line 1663, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.7/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-10-11 04:34:04,831 INFO    Thread-2  :30414 [wandb_init.py:init():664] run started, returning control to user process
2022-10-11 04:34:04,831 INFO    Thread-2  :30414 [wandb_run.py:_config_callback():992] config_cb None None {'img_feature_path': 'data/features/visualbert/', 'train_csv_path': 'data/splits/random/memotion_train.csv', 'val_csv_path': 'data/splits/random/memotion_val.csv', 'model_type': 'visualbert', 'model_path': 'uclanlp/visualbert-nlvr2-coco-pre', 'eval_step': 100, 'amp': True, 'output_dir': './tmp', 'checkpoint_step': 100, 'random_seed': 42, 'resume_training': False, 'semi_supervised': False, 'use_sweep': True}
2022-10-11 04:34:05,204 INFO    Thread-2  :30414 [wandb_run.py:_config_callback():992] config_cb None None {'output_hidden': 768}
2022-10-11 04:34:11,707 INFO    Thread-2  :30414 [wandb_run.py:finish():1539] finishing run vietnguyen/meme_experiments/gtgb5bqw
2022-10-11 04:34:11,708 INFO    Thread-2  :30414 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-10-11 04:34:11,710 INFO    Thread-2  :30414 [wandb_run.py:_restore():1674] restore
2022-10-11 04:34:11,715 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:34:11,822 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:34:11,979 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:34:12,081 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10140
}

2022-10-11 04:34:12,182 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,284 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,386 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,488 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,590 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,691 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:12,793 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:13,304 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}

2022-10-11 04:34:13,659 INFO    Thread-2  :30414 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10140
  total_bytes: 10140
}
local_info {
}

2022-10-11 04:34:15,344 INFO    Thread-2  :30414 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-10-11 04:34:15,344 INFO    Thread-2  :30414 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-10-11 04:34:15,345 INFO    Thread-2  :30414 [wandb_run.py:_footer_sync_info():2822] logging synced files
